src/server/src/main/java/io/cassandrareaper/service/RepairRunner.java

```diff
@@ -96,6 +100,9 @@ public RepairRunner(AppContext context, UUID repairRunId) throws ReaperException
     parallelRanges = getParallelRanges(
         parallelRepairs,
         Lists.newArrayList(Collections2.transform(repairSegments, segment -> segment.getTokenRange())));
+
+    metricNameForMillisSinceLastRepair = metricName("millisSinceLastRepair", keyspace);
+    metricNameForRepairProgress = metricName("repairProgress", keyspace);
```

Comment:
these names are not unique to the repair unit. 
there's no indication of cluster or tables involved. 
and it could also be future-proofed against multiple repair runners running in parallel…